.github/workflows/mavenCi.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+
+
+jobs:
+  job1:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+          cache: maven
+      - name: Build
+        run: |
+          mvn package -B clean verify --file pom.xml
+  
+
+  job2:
+    needs: job1
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'corretto'
+          cache: maven
+      - name: Analyze
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}  #Added new Repository Secret in GitHub
+        run: mvn sonar:sonar
+          -Dsonar.host.url="https://sonarcloud.io"
+          -Dsonar.organization=daviddamke
+          -Dsonar.projectKey=DavidDamke_KeepTime -X
+          -Dsonar.java.binaries=.
+          -Dsonar.qualitygate.wait=true
+      - name: Upload Build Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          # Artifact name
+          name: keeptime-1.2.0-SNAPSHOT-bin.zip
```

Comment:
the artifact does not contain the expected content (.jar + licenses). instead it has a folder with sonar files?

Reply:
The artifact now contains the expected content